doc/query.md

```diff
@@ -74,28 +74,28 @@ Basic querying of fields benefits but does _not require_ a JSON index. For examp
 Use the following methods to create a JSON index:
 
 ```
-ensureIndexed(List<Object> fieldNames, 
+ensureIndexed(List<Object> fieldNames,
               String indexName)
 ```
 
 Use either of the following methods to create a TEXT index:
 
 ```
-ensureIndexed(List<Object> fieldNames, 
-              String indexName, 
+ensureIndexed(List<Object> fieldNames,
+              String indexName,
               String indexType)
 
-// or 
+// or
 
-ensureIndexed(List<Object> fieldNames, 
-              String indexName, 
-              String indexType, 
+ensureIndexed(List<Object> fieldNames,
+              String indexName,
+              String indexType,
               Map<String, String> indexSettings)
-``` 
+```
 
 These indexes are persistent across application restarts as they are saved to disk. They are kept up to date as documents change; there's no need to call the `ensureIndexed(...)` method each time your applications starts, though there is no harm in doing so.
 
-The first argument, `fieldNames` is a list of fields to put into the index. The second argument, `indexName` is a name for the index. This is used to delete indexes at a later stage and appears when you list the indexes in the database.  The third argument, `indexType` defines what type of index to create.  Valid index types are `json` and `text`.  If not provided, the index type defaults to `json`.  The fourth argument, `indexSettings` is comprised of index parameters and their values.  Currently the only valid index setting is `tokenize` and it can only apply to a TEXT index.  If index settings are not provided for a TEXT index, the `tokenize` parameter defaults to the value `simple`.
+The first argument, `fieldNames` is a list of fields to put into the index. The second argument, `indexName` is a name for the index. This is used to delete indexes at a later stage and appears when you list the indexes in the database.  The third argument, `indexType` defines what type of index to create. If not provided, the index type defaults to `json`.  The fourth argument, `indexSettings` is comprised of index parameters and their values.  Currently the only valid index setting is `tokenize` and it can only apply to a TEXT index.  If index settings are not provided for a TEXT index, the `tokenize` parameter defaults to the value `simple`.
```

Comment:
In describing the third argument, I think we should specify that the index type defaults to JSON instead `json`.  We use the word TEXT to describe an index of type TEXT in the same paragraph, it only makes sense that we use JSON to describe the default for the index type.